sunraster/meta/spice.py

```diff
@@ -0,0 +1,106 @@
+
+from astropy.io import fits
+import astropy.units as u
+
+
+__all__ = ["SPICEMeta"]
+
+
+class SPICEMeta(fits.header.Header, SlitSpectrographMetaABC):
```

Comment:
I am not sure inherting from fits header is wise, it might be better long term to keep the header as an attribute rather than actually *this* object.